src/containerapp/azext_containerapp/_help.py

```diff
@@ -572,6 +572,15 @@
         az containerapp auth update -g myResourceGroup --name MyContainerapp --proxy-convention Standard
 """
 
+helps['containerapp env workload-profile set'] = """
+    type: command
+    short-summary: Create or update an existing workload profile in a Container Apps environment
```

Comment:
should you add deprecation_info here?

Reply:
Add in the deprecate info:
![image](https://github.com/Azure/azure-cli-extensions/assets/46143499/3a5bb748-5bc4-4d51-ab24-4c05f0d80567)